packages/tasks-gen/snippets-fixtures/document-question-answering/python/requests/0.hf-inference.py

```diff
@@ -12,8 +12,6 @@ def query(payload):
     return response.json()
 
 output = query({
-    "inputs": {
-		"image": "cat.png",
-		"question": "What is in this image?"
-	},
+    "image": "cat.png",
+    "question": "What is in this image?",
 })
```

Comment:
Is this input format correct?
(are you confident we do not need to wrap this under `inputs` ?)

Reply:
Good catch! It's indeed needed: https://huggingface.co/impira/layoutlm-document-qa?inference_provider=hf-inference&language=python&inference_api=true

Reply:
fixed in https://github.com/huggingface/huggingface.js/pull/1273/commits/df783f8a0c16a325717044b4260b40bcc494cb4b